scripts/mixins/families/amphiptere.lua

```diff
@@ -23,6 +23,34 @@ g_mixins.families.amphiptere = function(amphiptereMob)
         mob:setUntargetable(true)
         mob:setAnimationSub(1)
     end)
+
+    amphiptereMob:addListener('WEAPONSKILL_USE', 'REAVING_WIND_AURA', function(mobArg, target, actionId, tp, action)
+        local reavingWind   = 2431
+        local reavingWindKb = 2426
+
+        -- Amphipteres gain a temporary aura following the use of reaving wind.
+        if actionId == reavingWind then
+            mobArg:setAnimationSub(2)
+            -- Zirnitra spams a knockback while aura is active
+            mobArg:setLocalVar('auraEndTime', os.time() + 20)
+        end
+    end)
+
+    amphiptereMob:addListener('WEAPONSKILL_STATE_EXIT', 'SPAM_KNOCKBACK', function(mobArg, actionId)
+        local reavingWind   = 2431
+        local reavingWindKb = 2426
```

Comment:
the knockback being a hidden skill makes sense given the way knockback packets are made (atom0s gave me a rundown of it a few yrs back). I assume that ID was captured.